src/test/java/com/scopely/mapper/HashAndRange.java

```diff
@@ -1,17 +1,22 @@
 package com.scopely.mapper;
 
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAutoGeneratedKey;
 import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
 import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBRangeKey;
 import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
 import org.inferred.freebuilder.FreeBuilder;
+import org.jetbrains.annotations.Nullable;
 
 @FreeBuilder
 @JsonDeserialize(builder = HashAndRange.Builder.class)
 @DynamoDBTable(tableName = "hash_and_range")
 interface HashAndRange {
     @DynamoDBHashKey(attributeName = "hashKey")
     String getHashKey();
+    @Nullable
```

Comment:
You probably already tried this-- but it's annoying that we have to make this nullable to do this. I tried adding support for `Optional<String>` as the return type, but then there's no way we can check that we have a legal type for an autogenerated key at runtime-- we'd have to resort to something like annotation processing.